main.go

```
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"unicode/utf8"
+)
+
+func max(arr []string) (string, error) {
+	var maxIndex int = 0
+
+	if len(arr) == 0 {
+		errors.New("array is empty")
```

If you want to return an error, you should write `return` statement.

---

fixed. commit: b0fa3b15a1faedd249d886299cfa25f32fe372f1

---

fixed. commit: f800e42a48c9658197231d5e65b6dc77c8f07727